bin/import-previous-release-team-keyring

```diff
@@ -13,10 +13,7 @@ KEYS="9554F04D7259F04124DE6B476D5A82AC7E37093B \
       7937DFD2AB06298B2293C3187D33FF9D0246406D \
       1C050899334244A8AF75E53792EF661D867B9DFA"
 
-SERVERS="ha.pool.sks-keyservers.net
-         p80.pool.sks-keyservers.net:80 \
-         ipv4.pool.sks-keyservers.net \
-         keyserver.ubuntu.com
```

Comment:
I don't see anything online about these servers no longer existing. They all appear to still exist from what I can tell, and `keyserver.ubuntu.com` is reachable via browser.

Reply:
I didn't remove keyserver.ubuntu.com, that was the formatting of the PR.

The others are either offline and have no A records, do not respond to ping or have invalid certs.

Reply:
```
This service is deprecated. This means it is no longer maintained, and new HKPS certificates will not be issued. Service reliability should not be expected.

Update 2021-06-21: Due to even more GDPR takedown requests, the DNS records for the pool will no longer be provided at all.
```